lib/oli_web/live/delivery/remix_section.ex

```diff
@@ -501,6 +527,34 @@ defmodule OliWeb.Delivery.RemixSection do
     {:noreply, assign(socket, modal_assigns: modal_assigns)}
   end
 
+  def handle_event(
+        "HierarchyPicker.select",
+        %{"uuid" => uuid},
+        %{assigns: %{modal_assigns: %{active_tab: :all_pages}}} = socket
+      ) do
+    %{
+      modal_assigns:
+        %{
+          selection: selection,
+          table_model: table_model,
+          selected_publication: publication
+        } = modal_assigns
+    } = socket.assigns
+
+    item = Enum.find(table_model.rows, &(&1.uuid == uuid))
+
+    modal_assigns = %{
+      modal_assigns
+      | selection:
+          xor(
```

Comment:
I have no idea what this `xor` thing is hahaha, are you doing some kind of magic here?? 😂😂.

Reply:
Actually I just copied it from another place. The function is defined below, in line `755`. What it does is, if the tuple that is being added is already in the list, it removes it, if not, it adds it